### ISSUE TITLE
[Bug]: Soul Lava seems to function as a full block / blocks blocks being able to be placed in them or fall into them

### ISSUE THREAD

**Issue by SteveHeist** [AllTheMods/ATM-10#1449]
### Possible Fixes

Yes

### Modpack Version

2.6

### What happened?

Found some Soul Lava in the Other. Blocks cannot be placed in it or fall in it.
![2024-12-17_06 41 55](https://github.com/user-attachments/assets/23d68640-d8b1-4e3c-af75-4fc73336ea23)

**Comment by elowanut** [comment]
Yeah, this one's been around for a while. You can block the flow beforehand / very quickly, but you need to pick up the source block and wait for it to clear, once the flowing has begun.

**Comment by SteveHeist** [comment]
Yeah, it just... spawned this way.
